ts/nni_manager/training_service/reusable/environments/openPaiEnvironmentService.ts

```diff
@@ -96,7 +100,7 @@ export class OpenPaiEnvironmentService extends EnvironmentService {
         }
 
         const getJobInfoRequest: request.Options = {
-            uri: `${this.protocol}://${this.paiClusterConfig.host}/rest-server/api/v2/jobs?username=${this.paiClusterConfig.userName}`,
+            uri: `${this.protocol}://${this.paiClusterConfig.host}/rest-server/api/v2/jobs/${this.paiClusterConfig.userName}~${environment.envId}`,
```

Comment:
This change makes significant performance issue, and may trigger OpenPAI threshold to block API calls.

Reply:
reverted to use jobs API.